tests/test_utils.py

```diff
@@ -207,6 +213,59 @@ def test_quantiles(self):
             np.testing.assert_almost_equal(edges1, edges2, decimal=decimal)
 
 
+class ImageBasedTestCase(lsst.utils.tests.TestCase):
+    """A test case for testing sky position offsets for exposures."""
+
+    def test_fluxFromFootprint(self):
+        image = afwImage.Image(
+            np.arange(8100, dtype=np.int32).reshape(90, 90),
+            xy0=lsst.geom.Point2I(10, 12),
+            dtype="I"
+        )
+
+        radius = 3
+        spans = afwGeom.SpanSet.fromShape(radius, afwGeom.Stencil.CIRCLE, offset=(27, 30))
+        footprint1 = afwDetect.Footprint(spans)
+
+        # The extracted footprint should be the same as the product of the
+        # spans and the overlapped bow with the image
+        truth1 = spans.asArray() * image.array[15:22, 14:21]
+
+        radius = 3
+        spans = afwGeom.SpanSet.fromShape(radius, afwGeom.Stencil.CIRCLE, offset=(44, 49))
+        footprint2 = afwDetect.Footprint(spans)
+        truth2 = spans.asArray() * image.array[34:41, 31:38]
+
+        allFootprints = [footprint1, footprint2]
+        footprintSet = afwDetect.FootprintSet(image.getBBox())
+        footprintSet.setFootprints(allFootprints)
+
+        # check it can accept a footprintSet, and single and iterables of
+        # footprints
```

Comment:
Is it worth having an assertRaises for something that isn't one of those, just to make sure you didn't make your accept condition too broad?